src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -18,6 +18,24 @@
 
 LIBFUZZER_MAGIC_STRING = b"ERROR: libFuzzer"
 
+# These path constants point to identically-named managed DLLs, so it may seem like one
+# of them is redundant. This is false: the DLLs are _not_ identical, and each one links
+# platform-native code.
```

Comment:
Do we want to put the platform name into the DLL filename to make this clearer?

Reply:
This is naturally namespaced by container name (which does include the platform). Also, it turns out we don't need to make the distinction in this path constant, so we only need one. I've updated the comment accordingly.